pubsub/rabbitmq/metadata.go

```diff
@@ -15,6 +15,7 @@ package rabbitmq
 
 import (
 	"fmt"
+	"github.com/dapr/components-contrib/internal/utils"
```

Comment:
for the linter to pass you will need to remove dapr imports into a separate block below standard library things.

```
stdlib stuff
<blank>
dapr imports
```

or if you have third party imports also

```
stdlib stuff
<blank>
third party imports
<blank>
dapr imports
```

Reply:
Thanks, I sorted it now 👌